explore/src/main/scala/explore/tabs/ObsTabContents.scala

```diff
@@ -499,10 +499,15 @@ object ObsTabContents {
       // Shared obs conf (posAngle/obsTime)
       .useStateViewWithReuse(ObsConfiguration(PosAngle.Default, Instant.now))
       .useSingleEffect(debounce = 1.second)
+      // worker usage demo Delete soon
+      .useEffectOnMountBy((props, _, _, _, _, _, _) =>
+        props.ctx.worker.stream.evalMap(msg => IO.println(msg.data.toString)).compile.drain
```

Comment:
To get messages just get values from the stream

Reply:
Awesome